### ISSUE TITLE
from_filepath panics on non-existent file

### ISSUE THREAD

**Issue by commonquail** [aahancoc/tree_magic#3]
In tree_magic 0.2.0 the following panics from [unwrapping a `None`](https://github.com/aahancoc/tree_magic/blob/f72f65fd4ec6af05a63dacc0086a320139b535b7/src/lib.rs#L559).

```rust
let file = std::path::Path::new("does-not-exist");
tree_magic::from_filepath(file);
```

**Comment by casey** [comment]
`from_filepath()` should probably return a `std::io::Result`

**Comment by aahancoc** [comment]
Published 2.1 to fix issue. I apologize for the delay.
